[Term Entry] JavaScript Dates .getMilliseconds() (#2452)
Codecademy/docs: content/javascript/concepts/dates/terms/getMilliseconds/getMilliseconds.md

---
Title: '.getMilliseconds()'

> Description: 'Method of the Date object that returns the milliseconds of a date according to the local time.'

Reviewer: ```suggestion
Description: 'Returns the milliseconds of a date according to the local time.'
```

Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

> The **`.getMilliseconds()`** method of `Date` object returns the milliseconds of a date according to the local time.

Reviewer: ```suggestion
The **`.getMilliseconds()`** method of returns the milliseconds of a `Date` object according to the local time.
```

## Syntax

> ```js

Reviewer: ```suggestion
```pseudo
```

myDate.getMilliseconds();
```

The return value of `.getMilliseconds()` is an integer number between `0` and `999`, representing the milliseconds of a date according to the local time.

## Example

> To determine if it is half a second or not:

Reviewer: ```suggestion
The code below determines if the given value is equivalent to half a second, or not:
```

```js
const today = new Date('June 10, 2023 15:50:45:500');

if (today.getMilliseconds() === 500) {
  console.log('It is half a second.');
} else {
  console.log('It is not half a second.');
}

> // Output: It is half a second.

Reviewer: delete the comment and use a shell block-

```shell
It is half a second

```

## Codebyte Example

> In the following example, a new `Date` object `currentDateTime` is defined to store the current date and time. And `milliseconds` variable is declared to store the current milliseconds value using `.getMilliseconds()` method. Finally, `console.log` is used to print the values. Click on the **Run** button to see the output.

Reviewer: ```suggestion
In the following example, `currentDateTime` is defined to store the current date and time. The `.getMilliseconds()` method is used to extract the total milliseconds and both values are printed to the console.
```

Reviewer: The code example introductions should be very brief
